systemtest/src/main/java/io/strimzi/systemtest/clients/api/VerifiableClient.java

```diff
@@ -153,7 +153,7 @@ private void parseToList(String data) {
      * @return exit status of client
      */
     public boolean run() {
-        return runClient(60000, true);
+        return runClient(120000, true);
```

Comment:
We should be using `_` as thousands separator in literals this large, it's much easier to read.
```suggestion
        return runClient(120_000, true);
```

Reply:
Done